sdks/java/io/kafka/upgrade/src/main/java/org/apache/beam/sdk/io/kafka/upgrade/KafkaIOTranslation.java

```diff
@@ -221,6 +222,9 @@ public Row toConfigRow(Read<?, ?> transform) {
       fieldValues.put("redistribute", transform.isRedistributed());
       fieldValues.put("redistribute_num_keys", transform.getRedistributeNumKeys());
```

Comment:
should these other nullable fields, also have a null check?

Reply:
These are primitive booleans vs object Boolean for offset dedup (which was done so it can be unspecified for compatibility checks)